source_test.go

```diff
@@ -1,16 +1,18 @@
-package spanner_test
+package spanner
 
 import (
 	"context"
 	"testing"
 
-	spanner "github.com/conduitio-labs/conduit-connector-spanner"
+	testutils "github.com/conduitio-labs/conduit-connector-spanner/test"
 	"github.com/matryer/is"
 )
 
 func TestTeardownSource_NoOpen(t *testing.T) {
 	is := is.New(t)
-	con := spanner.NewSource()
+	con := NewSource()
 	err := con.Teardown(context.Background())
 	is.NoErr(err)
 }
+
+var singersTable testutils.SingersTable
```

Comment:
It's used in other prs in the snapshot and cdc iterators. I don't know, `source_test.go` feels like the common denominator.

It will also be used in the destination, so I don't know. Maybe, if I separate the connector into `source/` and `destination/` packages, then the `singersTable` could be in `source_test` and `destination_test`.

Reply:
For what it's worth, I'm not used to seeing a global variable declared that isn't a const in a file and being used in another. I'd rather define it in every method that's needed than have this one here. 🤷

Reply:
It's like a mini package. It's for testing files so I don't think the globality is much of an issue. I did the same in the mysql connector, and I saved quite a bit of boilerplate.

- If I didn't make it global, in every single test I would have to make an instance of the table.
- If they where just functions in test utils, I have to type `testutils.InsertUser` instead `usersTable.InsertUser`.

The later might be more correct now that I think about it. It should matter more if we splitted source and destination into 2 packages.